v_2_0_2/worker_template.go

```diff
@@ -295,6 +295,57 @@ coreos:
       --v=2"
       ExecStop=-/usr/bin/docker stop -t 10 $NAME
       ExecStopPost=-/usr/bin/docker rm -f $NAME
+    - name: k8s-proxy.service
```

Comment:
i still think its wrong as kubelet that is above has only 2 spaces before the `- name:` 
so the whole block might be shifted by 2 spaces

Reply:
@calvix Should be good now. PTAL.

Reply:
I thnik we still need to remove 2 space from `    - name: k8s-proxy.service` and `    - name: k8s-proxy-watcher.service` and we should be good :)

Reply:
Damnit, sorry this took a few goes :( Done.